scss/_patterns_matrix.scss

```diff
@@ -40,8 +40,10 @@
       }
     }
 
+    // TODO: Multiple redundant properties between this media query and the one
```

Comment:
Good catch, looks like an oversight from a long time ago when we removed one of the responsive breakpoints.

Probably not worth addressing knowing that this component is being phased out. Unless it starts to cause some bugs.